dynamic-rapidpro-workspace-settings/.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "parserOptions": {
-    "ecmaVersion": 2017
+    "ecmaVersion": 9
```

Comment:
You can use either the numerical versioning or by year. Either option works
https://eslint.org/docs/latest/user-guide/configuring/language-options

Reply:
Numerical versioning is cleaner.